src/borg/archiver/rinfo_cmd.py

```diff
@@ -40,6 +40,7 @@ def do_rinfo(self, args, repository, manifest, cache):
             {encryption}
             Cache: {cache.path}
             Security dir: {security_dir}
+            Storage quota: {used} used out of {storage_quota}
```

Comment:
line 36..40 all refer to repository properties, but 41..42 is about client-side stuff.

so i'ld move 43 to below 40.